### PR TITLE
ECCI-480/add-config-and-styles-for-related-news-items-on-article-pages

### PR DESCRIPTION
**Pull request by kurtisRogers-Nomensa**
### What have I done?
- Worked with Adrian to implement the content types and fields necessary for displaying the correct information
- Added templates for the new related news items content (accounting for anonymous users also)
- Added a slither of logic in the template to check if the user is authenticated else show sign in button - tested by logging in and out
- Added styles for the layout - heavily borrowed from what's already on the site to help keep things consistent
- Mobile first approach, looks good on all viewports
- Alos hid the related news title in article pages where content is restricted (where no related posts should display)

### CODE REVIEW

**Comment by Polynya** [comment]
Rebase to release/202311

File: themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig [hunk 1]
```diff
@@ -112,9 +112,11 @@
 
           {{ content|without('localgov_news_date', 'localgov_news_categories', 'localgov_news_related', 'sign_in') }}
 
-          {% if node.localgov_news_related.value %}
-            <h2 class="news-article__related-title">{{ 'Related news'|t }}</h2>
-            {{ content.localgov_news_related }}
+          {% if view_mode is not same as('full_anonymous') %}
```
Comment (Polynya) [inline themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig:115]: This condition shouldn't be necessary. This template is only for the full_anonymous display mode.

Comment (Polynya) [reply themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig:115]: I think I've already removed these lines on release/202311 branch.

File: config/default/core.entity_view_display.node.localgov_news_article.related_anonymous.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    module:
+      - localgov_news
+_core:
+  default_config_hash: T3PrBFQlZHLL6PIlyZqMvmSjh3fs5EiJsyXHC_Z1uMw
+id: node.localgov_news_article.related_anonymous
+targetEntityType: node
+bundle: localgov_news_article
+mode: related_anonymous
+content:
+  body:
```
Comment (Polynya) [inline config/default/core.entity_view_display.node.localgov_news_article.related_anonymous.yml:34]: Body, content_moderation_control, links, localgov_news_categories should be hidden

File: themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig [hunk 3]
```diff
@@ -112,6 +112,11 @@
 
           {{ content|without('localgov_news_date', 'localgov_news_categories', 'localgov_news_related', 'sign_in') }}
 
+          {% if node.localgov_news_related.value %}
```
Comment (Polynya) [inline themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig:115]: Remove these lines. Related content isn't displayed for anonymous users.

Comment (kurtisRogers-Nomensa) [reply themes/custom/essex/templates/content/node--localgov-news-article--full-anonymous.html.twig]: Done

**Comment by kurtisRogers-Nomensa** [comment]
Had a preliminary css conflict - resolved, should be good to go in!

File: config/default/core.entity_view_display.node.localgov_news_article.default.yml [hunk 4]
```diff
@@ -37,6 +37,11 @@ content:
     third_party_settings: {  }
     weight: 4
     region: content
+  content_moderation_control:
+    settings: {  }
+    third_party_settings: {  }
+    weight: 0
+    region: content
```
Comment (siliconmeadow) [inline config/default/core.entity_view_display.node.localgov_news_article.default.yml:44]: This shouldn't be there. We've been struggling to keep these from returning for months.

File: config/default/webform.webform.feedback_form.yml [hunk 5]
```diff
@@ -63,11 +63,9 @@ elements: |-
...
         Please do not enter any personal information in this space
         <div id="extension-host">&nbsp;</div>
-  have_you_liked_our_new_intranet_site_:
-    '#type': textfield
-    '#title': 'Have you liked our new Intranet site? '
-    '#description': 'Please give your feed back&nbsp;'
-    '#required': true
+    do_you_love_this_website_so_much_:
+      '#type': checkbox
+      '#title': 'Do you love this website so much?'
```
Comment (siliconmeadow) [inline config/default/webform.webform.feedback_form.yml:68]: Do you love this website so much?

Is this an official change?

**Review by siliconmeadow — CHANGES_REQUESTED** [review]

File: config/default/webform.webform.feedback_form.yml (hunk 5 above)
Comment (Polynya) [reply config/default/webform.webform.feedback_form.yml:68]: Sorry, it's not simple to keep the webform changes out of the config export, but these files need to be reverted.

Comment (kurtisRogers-Nomensa) [reply config/default/webform.webform.feedback_form.yml:68]: Nah I've removed it like 15 times 😆

**Review by siliconmeadow — APPROVED** [review]
LGTM

**Pull request merged**